nix/status-go/mobile/build.nix

```diff
@@ -47,7 +47,7 @@ in buildGoPackage {
       -ldflags="$ldflags" \
       -target=${concatStringsSep "," targets} \
       ${optionalString isAndroid "-androidapi=${platformVersion}" } \
-      ${optionalString isIOS "-iosversion=${platformVersion}" } \
+      ${optionalString isIOS "-iosversion=${platformVersion} -tags=netgo" } \
```

Comment:
Why was this flag added?

Reply:
to fix : https://github.com/golang/go/issues/58416 which was discovered in this PR https://github.com/status-im/status-mobile/pull/18564 and iOS build here were also failing with the same issue

Reply:
yeah sorry for this confusion. 
I usually never do this, I was trying to get CI pipelines to be green for `nixpkgs` upgrade 😿 
I did try to mention it in PR description though 😅